src/common/tests/CommonUtilsUT.cpp

```diff
@@ -19,7 +19,7 @@ using namespace std;
 class CommonUtilsTest : public ::testing::Test
 {
     protected:
-        const char* m_path = "~test.test";
+        const char* m_path = "/tmp/~test.test";
```

Comment:
Can we leverage `mktemp` here to make it more portable? See https://linux.die.net/man/1/mktemp